INVENTORY.md

```diff
@@ -1,106 +0,0 @@
-# Component Inventory
-
-Availability of [Spectrum](https://spectrum.adobe.com) components in [Spectrum CSS](https://opensource.adobe.com/spectrum-css/)
```

Comment:
This is not being updated regularly anymore and the script to do so was based on outdated dependency structures. If we want to bring this back, we'll need to find new ways to automate it.